kerckhoff/packages/serializers.py

```diff
@@ -172,3 +174,82 @@ def get_version_data(self, obj: Package):
     class Meta(PackageSerializer.Meta):
         fields = PackageSerializer.Meta.fields + ("version_data",)
         read_only_fields = PackageSerializer.Meta.read_only_fields + ("version_data",)
+
+class PackageInfoSerializer(serializers.ModelSerializer):
```

Comment:
I'd rather this be called PublicPackageSerializer

Reply:
Done.